.travis.yml

```diff
@@ -31,9 +32,13 @@ jobs:
       node_js:
         - 16
       script: yarn lint
-    - name: Test
+    - name: 'Test node 16'
```

Comment:
nodejs key on line 3 is defined as a list with 16 & 20.
are we sure that if we don't overwrite it here it will not use it as a matrix for this test job?

Reply:
By default it takes only the first one defined, yes.